doc/nrf/app_dev/device_guides/coprocessors/ug_hpf_softperipherals_comparison.rst

```diff
@@ -0,0 +1,116 @@
+﻿.. _ug_hpf_softperipherals_comparison:
+
+Introduction to High-Performance Framework and Soft Peripherals
+###############################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+:ref:`nrfxlib:soft_peripherals` and the :ref:`High-Performance Framework (HPF)<hpf_index>` both facilitate the creation of software-based peripherals.
+Choosing between these solutions depends on the specific requirements of your project and the resources available.
```

Comment:
Missing definition of what these two are. I feel an introduction should state that. You only state what their purpose is.

Reply:
Comment not applied. @masz-nordic , @lopeztel , please assist @annwoj and provide proper definitions.